src/CompactJson/MSBuild.CompactJsonResources.csproj

```diff
@@ -31,7 +31,14 @@
 	</ItemGroup>
 	<Target Name="IncludeDll" AfterTargets="AfterBuild">
 		<ItemGroup>
+			<None Include="$(PkgSystem_Text_Json)/lib/netstandard2.0/System.Text.Json.dll" Visible="False" Pack="True" PackagePath="buildTransitive;build" CopyToOutputDirectory="Always" CopyToPublishDirectory="Always" />
```

Comment:
This does not fix the bug. We are getting a new error message:

> (CompactJson target) -> 
  /Users/runner/.nuget/packages/msbuild.compactjsonresources/1.0.2-preview/buildTransitive/MSBuild.***.targets(20,3): error : Could not load file or assembly 'Microsoft.Bcl.AsyncInterfaces, Version=5.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' or one of its dependencies. [/Users/runner/work/1/s/TestApp/TestApp/TestApp/TestApp.csproj]

Reply:
You can also try using the 4.7.0 nuget as that package uses the same version of the dll that the msbuild on macos uses. This will prevent any other errors too.

Reply:
@mattleibow Thanks. It really works